src/query/v2/requests.hpp

```diff
@@ -496,12 +496,21 @@ struct UpdateEdgesResponse {
   bool success;
 };
 
+struct CommitRequest {
+  Hlc transaction_id;
+  Hlc commit_timestamp;
+};
+
+struct CommitResponse {
+  bool success;
+};
```

Comment:
We don't have 2PC right now, because if all the write requests succeed, then the transaction can be committed.

This is used when after potentially multiple write requests the query engine commits the changes.